config/config.go

```diff
@@ -154,3 +157,19 @@ func removeCurrentDirReference(dirs []string) {
 func isRelativePath(path string) bool {
 	return path[0] != '/' && path[0] != '~' && !windowsAbsolutePathPattern.MatchString(path)
 }
+
+// checkTodos trying find default TODO string and adding it if not exists
+func checkTodos(todos []string) []string {
```

Comment:
The name `checkTodos` doesn't really review the func's intent. One has to read the docstring or implementation to understand its purpose.

I suggest renaming this to something more understandable such as `addDefaultFormatIfMissing`

Reply:
Renamed